Sources/GetStartedApp/CloudantDatabaseManager.swift

```diff
@@ -70,52 +84,59 @@ class CloudantDatabaseManager: DatabaseManager {
 
   func addVisitors(user: [String: String]) -> Bool {
```

Comment:
I would prefer if this function either took a `User` instance as input or was:
```
func addVisitors(name: String) -> Bool {
```
It used to take a dictionary so it could be passed straight to couch but that isn't needed anymore. This would then simplify the code before.